static/app/views/eventsV2/data.tsx

```diff
@@ -50,6 +50,7 @@ export const WEB_VITALS_VIEWS: Readonly<Array<NewQuery>> = [
     projects: [],
     version: 2,
     range: '24h',
+    yAxis: ['epm()'],
```

Comment:
does this fix some bug?

Reply:
I added this because when adding a widget for the Web Vitals prebuilt query, the yAxis wasn't defined and saving from the query list page would default the y-axis to `count()`, but opening the page up changed the query to use `epm()`

I think the query should have `epm()` as the y-axis to be consistent